apps/pie-storybook/stories/pie-checkbox.stories.ts

```diff
@@ -13,10 +13,11 @@ type CheckboxStoryMeta = StoryMeta<CheckboxProps>;
 
 const defaultArgs: CheckboxProps = {
     ...defaultProps,
-    name: '',
+    name: 'name',
```

Comment:
This is needed for the form association to work in storybook, it took me a while to work out why it was broken!

Reply:
My bad, sorry. I wanted to update with my latest changes and forgot